server/src/dbschema.js

```diff
@@ -2,6 +2,12 @@ const db = require("./db");
 const Keygrip = require('keygrip');
 
 const createSQL = `
+
+CREATE TABLE IF NOT EXISTS images (
+  id VARCHAR(200) PRIMARY KEY,
+  image BYTEA
```

Comment:
Could be NOT NULL I think?

Also, seems like it should have a reference to its shot, and ON DELETE CASCADE